src/Documentation~/compatibility.md

```diff
@@ -0,0 +1,15 @@
+# Compatibility with `Mathf`
+
+Unity Mathematics isn't implemented in the same way as the [`UnityEngine.Mathf`](https://docs.unity3d.com/ScriptReference/Mathf.html) struct in the core Unity engine. Because of this, if your application relies on the specific behaviors of `Mathf`, it might not be possible to use Unity Mathematics in your project. Also, if your project uses `Mathf` functions that have no corresponding Unity Mathematics function, or your code uses external plug-ins that depend on Unity types such as `Vector3`, or `Matrix4x4`, then you shouldn't use Unity Mathematics.
+
+>[!IMPORTANT]
+>For performance reasons, if your project uses the Mono compiler, you should continue to use the mathematical operations in `Mathf`, rather than Unity Mathematics.
+
+## Porting `Mathf` code to Unity Mathematics
+
+If you want to migrate code from `UnityEngine` to `Unity.Mathematics`, you need to make the following changes first:
+
+* `Vector4` to `float4`
+* Multiplications on vector or matrix objects to component multiplication with the `*` operator.
```

Comment:
I think this point on multiplication needs to be changed. Currently, it makes it sound like you need to go through the effort to change your matrix multiplications to be compatible with componentwise multiplication, but what I'm trying to warn users about is that the multiplication operator is fundamentally different in Unity.Mathematics and is the number 1 pitfall people encounter when first using it.

I don't know the ideal way of saying this but my stab is: `Update any usages of operators involving matrices or vectors. For example, the Matrix4x4 multiplication operator implements matrix multiplication but the float4x4 multiplication operator implements componentwise multiplication`.